src/ape_accounts/_cli.py

```diff
@@ -157,6 +157,17 @@ def _import(cli_ctx, alias, import_from_mnemonic, custom_hd_path):
     )
 
 
+@cli.command(name="export", short_help="Unlock an account and export the private key")
+@ape_cli_context()
+@existing_alias_argument()
+def _export(cli_ctx, alias):
```

Comment:
Click will generate command name from the function name here, there's no need for the underscore (unlike `_import`) 
```suggestion
def export(cli_ctx, alias):
```

Reply:
ok perfect thanks!